components/layout/analytics.tsx

```diff
@@ -0,0 +1,72 @@
+import Script from "next/script";
+
+const zendeskKey = process.env.NEXT_PUBLIC_ZENDESK_CHAT_KEY;
+const gaId = process.env.NEXT_PUBLIC_GOOGLE_ANALYTICS;
+const gtmId = process.env.NEXT_PUBLIC_GOOGLE_GTM_ID;
+const hjId = process.env.NEXT_PUBLIC_HOTJAR_ID;
+const hjSv = process.env.NEXT_PUBLIC_HOTJAR_SV;
+
+export const Analytics = () => {
```

Comment:
can we separate the different analytics bits into separate components (in this file)?

Should make it easier to read that a big wall of script tags

Reply:
Done - Looks a bit better now